docs/guides/haiku_migration_guide.rst

```diff
@@ -861,4 +861,47 @@ as we are using ``5`` layers.
                   kernel: (5, 64, 64),
               },
           },
-      })
\ No newline at end of file
+      })
+
+Raw Haiku functions vs Flax modules
```

Comment:
Maybe "top-level" is a better description for what we are trying to express here? I think in practice in Haiku people only code the "main module" as a function, submodules are proper `hk.Module`s.

```suggestion
Top-level Haiku functions vs top-level Flax modules
```